ROOTFS/etc/waggle/sanity/fatal/core_nvme_emmc.test

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# nx core tests
+pf="NVME and eMMC tests"
+
+echo "$pf: Begin"
+
+# properly mounted tests 
+if [ "$(df -h | grep "/media/root-ro")" = "" ];
+then
+        echo "$pf: RO partition not mounted FAIL"
+        exit 1
+fi
+
+if [ "$(df -h | grep "/media/root-rw")" = "" ];
+then
+        echo "$pf: RW partition not mounted FAIL"
+        exit 2
+fi
+
+if [ "$(df -h | grep "/media/plugin-data")" = "" ];
+then
+        echo "$pf: plugin-data not mounted FAIL"
+        exit 3
+fi
+
+# fsck pass
+if [ "$(cat /run/initramfs/fsck.log | grep "/dev/mmcblk0p1: clean")" = "" ];
+then
+        echo "$pf: incorrect fsck pass on nx-core FAIL"
+        exit 4
+fi
+
```

Comment:
I would like you to add a comment about checking the fsck of `nvme0n1p3  259:3    0   256G  0 part /media/root-rw` in the future.  a TODO comment.